frontend/src/PublishedPostsTile.tsx

```diff
@@ -0,0 +1,146 @@
+	H248.082z"/>
+                <path id="f" fill="#FFFFFF" d="M182.409,262.307v-99.803h33.499l5.016-38.895h-38.515V98.777c0-11.261,3.127-18.935,19.275-18.935
+	l20.596-0.009V45.045c-3.562-0.474-15.788-1.533-30.012-1.533c-29.695,0-50.025,18.126-50.025,51.413v28.684h-33.585v38.895h33.585
+	v99.803H182.409z"/>
+            </svg>
+        }
+        else if (platformId === 2) {
+            return <svg width="1.5em" height="1.5em" version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" x="0px" y="0px"
+                viewBox="0 0 210 279.9">
+                <path fill="#157DC3" d="M165.5,268.2H94.3l-1.5-0.1c-48.4-4.4-80.8-40.8-80.5-90.3V41.8c0-17.7,14.3-32,32-32s32,14.3,32,32v47.2
+                   l92.9,0.9c17.7,0.2,31.9,14.6,31.7,32.3c-0.2,17.6-14.5,31.7-32,31.7c-0.1,0-0.2,0-0.3,0L76.3,153v24.9
+                   c-0.1,22.7,14.1,25.6,21,26.3h68.2c17.7,0,32,14.3,32,32S183.2,268.2,165.5,268.2z"/>
+            </svg>
+        }
+        else {
+            return <svg version="1.0" xmlns="http://www.w3.org/2000/svg"
+                width="1.5em" height="1.5em" viewBox="0 0 1280.000000 1280.000000">
+                <g transform="translate(0.000000,1280.000000) scale(0.100000,-0.100000)"
+                    fill="#d94f4f">
+                    <path d="M1327 11473 l-1327 -1328 1872 -1872 1873 -1873 -1873 -1873 -1872
+           -1872 1327 -1328 1328 -1327 1872 1872 1873 1873 1873 -1873 1872 -1872 1328
+           1327 1327 1328 -1872 1872 -1873 1873 1873 1873 1872 1872 -1327 1328 -1328
+           1327 -1872 -1872 -1873 -1873 -1873 1873 -1872 1872 -1328 -1327z"/>
+                </g>
+            </svg>;
+        }
+    }
+
+    /** Fire an event to change the selected post. Lets all listeners on the event bus (e.g. PublishMessageTile.tsx) know to update accordingly. */
+    private selectPost(post: PublishedPost) {
+        this.state.eventBus.fireEvent(EventBusEventType.SELECTED_POST_CHANGED, post);
+    }
+
+    public render() {
+        const posts = this.state.posts.sort(
+            (p1: PublishedPost, p2: PublishedPost) => p2.created.getTime() - p1.created.getTime())
+            .map(elem => {
+                return (
+                    <tr onClick={() => this.selectPost(elem)}>
+                        <td>
+                            <Moment format="YYYY-MM-DD">{elem.created}</Moment>
+                        </td>
+                        <td>{this.getSocialmediaIcon(elem.platformId)}</td>
+                        <td>{elem.creatorName}</td>
+                        <td>{elem.text}</td>
+                    </tr>
+                );
+            });
+
+        const updating = this.state.updating ? "btn-align-top-right btn-icon crossRotate" : "btn-align-top-right btn-icon";
+
+        return (
+            <div className="container double-container">
+                <div className="box-header">
+                    Published Posts
+                    <div
+                        className={updating}
```

Comment:
Maybe the variable name could indicate that it holds `className`s. And as far as I can see, there's just an additional class, the other two are identical. Maybe we could do something like this:
```suggestion
        let classUpdating = ["btn-align-top-right", "btn-icon"]
        if (this.state.updating) {
            classUpdating.push("crossRotate");
        }

        return (
            <div className="container double-container">
                <div className="box-header">
                    Published Posts
                    <div
                        className={updating.join(" ")}
```

or we could give [styled components](https://github.com/styled-components/styled-components) a try.

Reply:
done